shared/utils/domains.ts

```diff
@@ -82,6 +61,10 @@ export function isCustomSubdomain(hostname: string) {
   return true;
 }
 
+export function getCookieDomain(domain: string) {
+  return env.SUBDOMAINS_ENABLED ? stripSubdomain(domain) : domain;
```

Comment:
I think we should remove `SUBDOMAINS_ENABLED` soon, just tie it to hosted deployment

Reply:
yeah they do seem to be 1:1 paired